audio-worklet/noisegate-test.html

```diff
@@ -62,7 +62,7 @@
         // The signal should be all 1's prior to timeToThreshold to simplify
         // the math that relates the closing index to the timeConstant.
         let gain = new GainNode(context, {gain : 1});
-        let constantSource = new ConstantSourceNode(context, {offset: 1});
+        let constantSource = new ConstantSourceNode(context, {gain: 1});
```

Comment:
It was a style comment.  You should know that `ConstantSourceNode` has no gain option, so replacing `offset` with `gain` is wrong.  You should have made both line 64 and 64 look like "foo: x" or "foo : x".  The former seems like the typical style.